app/components/blacklight/header_component.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Blacklight
+  class HeaderComponent < Blacklight::Component
+    def initialize(blacklight_config:)
+      @blacklight_config = blacklight_config
+    end
+
+    attr_reader :blacklight_config
+
+    def call
+      render(top_bar) + render(search_bar)
```

Comment:
Maybe `top_bar` and `search_bar` could be slots with default values, like we've done in other components? It's a little easier to change them out that way.